lib/auth/userpreferences/userpreferencesv1/service_test.go

```diff
@@ -64,7 +64,7 @@ func TestService_GetUserPreferences(t *testing.T) {
 						DefaultTab:            userpreferencesv1.DefaultTab_DEFAULT_TAB_ALL,
 						ViewMode:              userpreferencesv1.ViewMode_VIEW_MODE_CARD,
 						LabelsViewMode:        userpreferencesv1.LabelsViewMode_LABELS_VIEW_MODE_COLLAPSED,
-						AvailableResourceMode: userpreferencesv1.AvailableResourceMode_AVAILABLE_RESOURCE_MODE_ALL,
+						AvailableResourceMode: userpreferencesv1.AvailableResourceMode_AVAILABLE_RESOURCE_MODE_NONE,
```

Comment:
Should we not add a separate test for `AvailableResourceMode_AVAILABLE_RESOURCE_MODE_NONE ` rather than replacing the `AvailableResourceMode_AVAILABLE_RESOURCE_MODE_ALL` test?

Same goes for other tests below

Reply:
in the context of these tests, its just testing the functionality for updating `AvailableResourceMode` property when upserting user prefs (and using the _new_ default of `NONE` instead of `ALL`)